src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-attribute.mdx

```diff
@@ -0,0 +1,722 @@
+---
+title: Set a custom attribute
```

Comment:
@ally-sassman In the nav, it's "Create custom attribute". I think it should match this title